src/modules/ordersTable/pure/OrdersTableContainer/TableGroup.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react'
+
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { Currency, Price } from '@uniswap/sdk-core'
+
+import { transparentize } from 'polished'
+import styled from 'styled-components/macro'
+
+import { OrderStatus } from 'legacy/state/orders/actions'
+
+import { PendingOrderPrices } from 'modules/orders/state/pendingOrdersPricesAtom'
+import { BalancesAndAllowances } from 'modules/tokens'
+
+import { OrderRow } from './OrderRow'
+import * as styledEl from './OrderRow/styled'
+import { OrdersTablePagination } from './OrdersTablePagination'
+import { LimitOrderActions } from './types'
+import { getOrderParams } from './utils/getOrderParams'
+
+import { ORDERS_TABLE_PAGE_SIZE } from '../../const/tabs'
+import { OrderTableGroup } from '../../utils/orderTableGroupUtils'
+
+const GroupBox = styled.div``
+
+const Pagination = styled(OrdersTablePagination)`
+  background: ${({ theme }) => transparentize(0.91, theme.text1)};
+  margin: 0;
+  padding: 10px 0;
+`
+
+export interface TableGroupProps {
+  item: OrderTableGroup
+  prices: PendingOrderPrices | undefined | null
+  spotPrice: Price<Currency, Currency> | undefined | null
+  isRateInverted: boolean
+  isOpenOrdersTab: boolean
+  isRowSelectable: boolean
+  isRowSelected: boolean
+  orderActions: LimitOrderActions
+  chainId: SupportedChainId
+  balancesAndAllowances: BalancesAndAllowances
+}
+
+export function TableGroup(props: TableGroupProps) {
+  const {
+    item,
+    prices,
+    spotPrice,
+    isRateInverted,
+    isOpenOrdersTab,
+    isRowSelectable,
+    isRowSelected,
+    orderActions,
+    chainId,
+    balancesAndAllowances,
+  } = props
+
+  const { parent, children } = item
+
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(true)
+  const [currentPage, setCurrentPage] = useState<number>(1)
+
+  const childrenLength = children.length
+  const step = currentPage * ORDERS_TABLE_PAGE_SIZE
+  const childrenPage = children.slice(step - ORDERS_TABLE_PAGE_SIZE, step)
+
+  const isParentSigning = parent.status === OrderStatus.PRESIGNATURE_PENDING
+
+  const commonProps = {
+    isRowSelectable,
+    isOpenOrdersTab,
+    spotPrice,
+    prices,
+    isRateInverted,
+    orderActions,
+  }
+
+  return (
+    <GroupBox>
+      <OrderRow
+        {...commonProps}
+        key={parent.id}
+        isRowSelected={isRowSelected}
+        order={parent}
+        orderParams={getOrderParams(chainId, balancesAndAllowances, parent)}
+        onClick={() => orderActions.selectReceiptOrder(parent)}
```

Comment:
not from this PR, but the action `selectReceiptOrder` was confusing `showReceipt`

Also, I think more than `onClick` the property should be called, `showReceipt` too since there's more click events in the row?

I also wonder if we should pass and optional callback `showChildren` (with `() => setIsCollapsed((state) => !state)`) that if present, means its a parent row. This way we can show the details when they click on the ROW (not just in the status) --> not sure if everyone will like this idea, but for me makes sense (feel free to ignore if you think is confusing to collapse/show by clicking in the row instead of showing the receipt)